packages/react/src/components/TableOfContents/TableOfContents.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import root from 'window-or-global';
+import { TABLE_OF_CONTENTS } from '../../internal/FeatureFlags';
```

Comment:
@sannminwin need to update this one: `DDS_TOC`

Reply:
updated.